modules/swagger-parser/src/test/java/io/swagger/parser/OpenAPIParserTest.java

```diff
@@ -25,9 +26,11 @@
 
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertNull;
 import static org.testng.Assert.assertTrue;
 
 public class OpenAPIParserTest {
+
```

Comment:
what is this change for? possibly revert changes for this file

Reply:
yes I will